pkg/gofr/http/request.go

```diff
@@ -179,3 +179,8 @@ func (r *Request) bindBinary(raw interface{}) error {
 
 	return nil
 }
+
+// SetPathParam sets URL path parameters for the given HTTP request.
+func SetPathParam(r *http.Request, prm map[string]string) *http.Request {
+	return mux.SetURLVars(r, prm)
+}
```

Comment:
I'm unsure about the function purpose, while mux one provides it

Maybe it's about making this easier to find what to do, but the comment could explain why it's a simple wrapper/alias